shub/schedule.py

```diff
@@ -43,17 +43,24 @@
 @click.command(help=HELP, short_help=SHORT_HELP)
 @click.argument('spider', type=click.STRING)
 @click.option('-a', '--argument',
-              help='spider argument (-a name=value)', multiple=True)
+              help='Spider argument (-a name=value)', multiple=True)
 @click.option('-s', '--set',
-              help='job-specific setting (-s name=value)', multiple=True)
-def cli(spider, argument, set):
+              help='Job-specific setting (-s name=value)', multiple=True)
+@click.option('-p', '--priority', default=2,
```

Comment:
Let's add int type check for `priority`/`units` options?

Reply:
`int` is assumed because of the default value, but I agree that _explicit is better than implicit_